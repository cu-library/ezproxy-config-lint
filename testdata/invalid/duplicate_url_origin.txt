Option UTF16
Title Siku Quanshu
URL http://skqs.yourlib.org
DJ skqs.yourlib.org
Option NoUTF16

Option UTF16
Title Siku Quanshu Two
URL http://skqs.yourlib.org
DJ skqs.yourlib.org
Option NoUTF16
