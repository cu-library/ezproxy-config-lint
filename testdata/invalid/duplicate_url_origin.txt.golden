testdata/invalid/duplicate_url_origin.txt:9: URL http://skqs.yourlib.org ← Origin already seen at "testdata/invalid/duplicate_url_origin.txt:3" (L2002)
